### PR TITLE
Show total time for project

### LINKED ISSUE #5206: Add hours to project list
## Detailed Description
Add another column `Hours` to the projects list in the backend.
@fm3 can you comment whether this would be feasible perf-wise?

## Context
For billing purposes, we need to know the total number of hours per project.

### PR DESCRIPTION
**Pull request by fm3**
Adds the project total time to the project list

### URL of deployed dev instance (used for testing):
- https://projecttime.webknossos.xyz

### Steps to test:
- create tasks, trace some
- tracing time should show in project list

### Issues:
- fixes #5206 

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Ready for review

### CODE REVIEW

File: app/utils/SitemapWriter.scala [hunk 1]
```diff
@@ -18,7 +18,7 @@ case class SitemapURL(url: String,
 
 class SitemapWriter @Inject()(publicationDAO: PublicationDAO, wkConf: WkConf)(implicit ec: ExecutionContext)
     extends FoxImplicits {
-  private val proxyURLs = wkConf.Proxy.routes.map(SitemapURL(_))
+  private val proxyURLs = wkConf.Proxy.routes.filter(!_.contains("*")).map(SitemapURL(_))
```
Comment (fm3) [inline app/utils/SitemapWriter.scala:21]: Note for reviewer: This is unrelated, slipped in here. We noticed that the wildcard feature of the URL proxy conflicts with listing the proxy URLs in the sitemap. If the feature is used, those should not be listed (as with the wildcard they are not real URLs)

**Comment by fm3** [comment]
(snapshot tests still fail currently, will refresh snapshots with new field here after #4911 is merged

File: frontend/javascripts/admin/project/project_list_view.js [hunk 2]
```diff
@@ -288,6 +288,18 @@ class ProjectListView extends React.PureComponent<PropsWithRouter, State> {
                 width={200}
                 sorter={Utils.compareBy(typeHint, project => project.numberOfOpenAssignments)}
               />
+              <Column
+                title="Time [h]"
```
Comment (philippotto) [inline frontend/javascripts/admin/project/project_list_view.js:292]: Do you think "Time" is self-explanatory? Otherwise, we could add a tooltip saying "Total time spent annotating for this project".

**Review by philippotto — COMMENTED** [review]
Front-end looks good :+1: Didn't test, though, as the CI didn't pass.

**Review by youri-k — APPROVED** [review]
Backend LGTM :tada:
